bazaarvoice/emodb
# Adding streaming api document

## Github Issue #

n/a

## What Are We Doing Here?

Updating document for streaming api

## How to Test and Verify

1. Check out this PR
2. Follow the direction in the command line
3. Make sure it works.
